test/CoWSwapEthFlow.t.sol

```diff
@@ -416,6 +416,26 @@ contract OrderDeletion is EthFlowTestSetup {
         ethFlow.deleteOrder(order.data);
     }
 
+    function testCanDeleteManyOrders() public {
+        address owner = address(0x424242);
+        address executor = address(0x1337);
+        EthFlowOrder.Data[] memory orderArray = new EthFlowOrder.Data[](2);
+        orderArray[0] = dummyOrder();
+        orderArray[0].validTo = uint32(block.timestamp) - 1;
+        OrderDetails memory order1 = orderDetails(orderArray[0]);
+        mockOrderFilledAmount(order1.orderUid, 0);
+        createOrderWithOwner(order1, owner);
+        orderArray[1] = dummyOrder();
+        orderArray[1].validTo = uint32(block.timestamp) - 1;
+        orderArray[1].sellAmount = orderArray[1].sellAmount + 1;
+        OrderDetails memory order2 = orderDetails(orderArray[1]);
+        createOrderWithOwner(order2, owner);
+        mockOrderFilledAmount(order2.orderUid, 0);
+
+        vm.prank(executor);
+        ethFlow.deleteManyOrders(orderArray);
```

Comment:
Here we could also test that both orders were successfully cancelled, for example by confirming that the initial owner balance is the same as the final owner balance.